core/src/test/java/org/elasticsearch/bootstrap/BootstrapCheckTests.java

```diff
@@ -197,4 +199,15 @@ public void testEnforceLimits() {
         assertTrue(BootstrapCheck.enforceLimits(settings));
     }
 
+    public void testMinMasterNodes() {
+        BootstrapCheck.Check check = new BootstrapCheck.MinMasterNodesCheck();
+        assertTrue(check.check(Settings.EMPTY));
+        assertFalse(check.check(
+            Settings.builder().put(ElectMasterService.DISCOVERY_ZEN_MINIMUM_MASTER_NODES_SETTING.getKey(), 1).build()
+        ));
+        expectThrows(RuntimeException.class, () ->
+            BootstrapCheck.check(
```

Comment:
I personally wanted to check "outside behavior" without really know what the thing does. I give a setting with no min master nodes and strictness is enforced -> boom.  I don't care how. Here too -I don't mind much. Can change if you prefer.